Modules/Battery/settings.swift

```diff
@@ -110,7 +110,7 @@ internal class Settings: NSView, Settings_v {
     }
     
     @objc private func changeUpdateInterval(_ sender: NSMenuItem) {
-        if sender.title == "Disabled" {
+        if sender.title == LocalizedString("Disabled") {
```

Comment:
Sorry, but the saving key cannot be translated. It must be constant.

Reply:
Thank you for your attention! I just made a new commit to keep the `lowLevelNotification` and saving term `Battery_lowLevelNotification` with a constant value "Disabled".